src/providers/types.ts

```diff
@@ -1,2 +1,2 @@
-export type RPCProvider = "INFURA" | "ALCHEMY";
+export type RPCProvider = "INFURA" | "ALCHEMY" | "DRPC";
```

Comment:
Is this missing `INFURA_DIN` here?

Reply:
Yeah, I think so: https://github.com/across-protocol/sdk/pull/706/commits/5aadadf9c0a04a7cc349a53d1be42aeb40b48cd5

Reply:
My mistake on that fwiw - I added INFURA_DIN to the type predicate but not the actual type :facepalm: